db/lnbolt/peerdb.go

```diff
@@ -126,7 +132,11 @@ func (pdb *peerboltdb) GetPeerInfos() (map[lncore.LnAddr]lncore.PeerInfo, error)
 				return err2
 			}
 
-			ka := lncore.LnAddr(string(k))
+			ka, err2 := lncore.ParseLnAddr(string(k))
```

Comment:
`ka`? (also nit, but could be just `err`)

Reply:
"`k` as `a`ddress"

And `go vet` was complaining something about shadowing so I made it a different name.

Reply:
oh, `go vet` stuff, cool